vmodules/util/interfaces/BotConfig.ts

```diff
@@ -17,5 +29,69 @@ export default interface BotConfig {
   },
   api: {
     port: number
-  }
+  },
```

Comment:
From here downward is *very* messy. Ideally i'd like to split this section into its own interface (can be in the same file), then join the two and export the joined interface.

Reply:
uh... did you *combine* the bot and guild configs for this??? why???????

Reply:
the bot and guild configs are separate files for a reason...

the bot config only has properties that are used by the bot itself internally.

the guild config is the template for per-guild configs.